packages/zapp/console/src/components/Entities/VersionDetails/EntityVersionDetailsContainer.tsx

```diff
@@ -39,9 +43,9 @@ const useStyles = makeStyles((theme: Theme) => ({
   versionsContainer: {
     display: 'flex',
     flex: '0 1 auto',
-    height: '40%',
+    height: ({ resourceType }) => (resourceType === ResourceType.LAUNCH_PLAN ? '100%' : '40%'),
```

Comment:
currently in "All Versions" sections we have a radio buttons, however switching between versions doesn't have any effect.
@jsonporter Does it make sense to hide the radiobutton column for now?
Not sure if we plan to show expected inputs per each version and fixed inputs as in Figma, as theoretically they can differ in different launch plan versions.
Current view:
<img width="1143" alt="Screen Shot 2022-06-26 at 7 14 39 PM" src="https://user-images.githubusercontent.com/55718143/175827100-690a9439-97d9-4ed9-a6f0-129c79ae453f.png">
Figma:
<img width="694" alt="Screen Shot 2022-06-26 at 7 45 45 PM" src="https://user-images.githubusercontent.com/55718143/175827211-9f6ae075-3c2e-475d-876f-2d1899cd68f2.png">